src/util/std_expr.h

```diff
@@ -3450,6 +3450,19 @@ class function_application_exprt:public exprt
     operands().resize(2);
   }
 
+  explicit function_application_exprt(const typet &_type):
+    exprt(ID_function_application, _type)
+  {
+    operands().resize(2);
+  }
+
+  function_application_exprt(
+    const symbol_exprt &_function, const typet &_type):
+      function_application_exprt(_type)
```

Comment:
Please add `// NOLINT(runtime/explicit)` to this line to silence the linter on this invocation of the constructor from the two-parameter one.